### DISCUSSION TITLE
Is it possible to influence the order of the entries on the website

### DISCUSSION THREAD

**Discussion by Rubber1Duck** [syssi/esphome-soyosource-gtn-virtual-meter#170] [Q&A] [answered]
as the headline:
Is it possible to influence the order of the entries on the website? It seems this is fully random .....
Thanks in advance for any answer

**Comment by syssi** [comment, marked as answer]
No. The order is alphabetically and cannot be changed without renaming the entities as far as I know.

**Reply by Rubber1Duck** [reply]
Thanks a lot!

**Comment by swat2go** [comment]
I guess you both classically talked about different things and even I may be wrong as all content is NOT EXPLICITLY clear. 
just ask elon kindly to change the order  - that's it!

I assume the opener talked about the internal webinterface of the ESP module itself whereas Syssi talks about homeassistant due to refering to entities which are imho nowhere in the ESP or github code. Right? It could also be possible to talk about open.... webinterface as long as no one talks clear or we can watch her/his thoughts.  

I am also hassling where to find which value when opening the direct webinterface of the esp.  the regulation is also a big optimizing / failure point to as iam tight on solar harvesting.

**Reply by syssi** [reply]
No. I talked about the web interface too. In the past the tables at the web interface wasn't customizable. This has changed over time and the order of the entities can be changed now. See https://esphome.io/components/web_server.html#entity-sorting
